project/src/components/app/app.tsx

```diff
@@ -1,7 +1,23 @@
 import Main from 'pages/main/main';
+import NotFoundScreen from 'pages/not-found-screen/not-found-screen';
+import {Route, Routes} from 'react-router-dom';
+import {AppRoute} from 'const';
 
 function App(): JSX.Element {
-  return <Main />;
+  return (
+    <Routes>
+      <Route path="/">
+        <Route
+          index
+          element={<Main />}
```

Comment:
не знаю в чем точно отличие, но обычно используют component, а не element

Reply:
а ну и роут в роуте быть не должен
```suggestion
          <Route
            path={AppRoute. Main}
            component ={<Main />}
         />
```

Reply:
<img width="712" alt="image" src="https://user-images.githubusercontent.com/88372125/186971324-ad4704fb-7b85-459a-8be3-dff4f5ec1939.png">

https://habr.com/ru/company/kts/blog/598835/

Reply:
огонь
спасибо, тоже чот новое узнал)